packages/scheduler-prisma/src/helpers.ts

```diff
@@ -19,22 +19,28 @@ export const isHealthy = (heartbeat: number, timeout: number) =>
   new Date().getTime() - timeout < heartbeat;
 
 const getValidRule = (recurrence: string, timezone?: string) => {
-  const isICalRule = recurrence.includes('DTSTART');
+  const isICalRule = recurrence.startsWith('DTSTART');
   if (isICalRule) return recurrence;
 
-  let derivedTimezone = timezone ?? 'Australia/Sydney';
+  let derivedTimezone = timezone ?? 'UTC';
```

Comment:
for computeNextRun and computeNextRuns methods they have UTC as default (line 54 and line 93)